cfbs/git.py

```diff
@@ -85,14 +85,18 @@ def git_init(user_name=None, user_email=None, description=None):
             f.write(description + "\n")
 
 
-def git_commit(commit_msg, edit_commit_msg=False, scope="all"):
+def git_commit(
+    commit_msg, edit_commit_msg=False, user_name=None, user_email=None, scope="all"
+):
     """Create a commit in the CWD Git repository
 
     :param commit_msg: commit message to use for the commit
     :param scope: files to include in the commit or `"all"` (`git commit -a`)
     :type scope: str or an iterable of str
     :param edit_commit_message=False: whether the user should be prompted to edit and
                             save the commit message or not
+    :param user_name=None: override git config user name
```

Comment:
I don't think the `=None` part is necessary here (and maybe even not supported by Sphinx?).